src/frontend/posts_page/components/PostBlock.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+
+interface MyProps {
+  post: Post
+}
+
+interface Post {
+  name: string,
+  time: string,
+  detail: string
+}
+
+interface MyState {
+  like: number,
+  liked: boolean
+}
+
+
+class PostBlock extends React.Component<MyProps, MyState> {
+  constructor(props: MyProps) {
+    super(props);
+    this.state = {
+      like: 0,
+      liked: false
+    };
+  }
+
+  markLike = () => {
+    if(this.state.liked) {
+      this.setState({like: this.state.like - 1});
+    } else {
+      this.setState({like: this.state.like + 1});
+    }
+    this.setState({liked: !this.state.liked});
+  };
+
+
+  render() {
+    const image = this.props.post.name === "Bain" ? <img className="inserted-image" src={require("./sampleImage.jpg")} alt="sampleImage" /> : '';
```

Comment:
This is just for Saturday review. As I am not sure how things would be saved after we learn about database stuffs, I just want something can display and easy to remove in the future.

Reply:
That's the point. For frontend, we don't want to care about the backend details. It will just take a parameter and renders the desired elements. What I did is to put all the temporary stuff in one place. (In App.tsx) So we don't need to modify all the components in the future.

Reply:
Yes, so that's why I just add one simple line here to display the image. It will be removed when I have a clear idea about backend stuffs.

Reply:
Just to clarify, I put a boolean statement here because I only want the image to display on some certain posts. Thus, it actually doesn't matter.